src/engine/Test_dataflow_tainting.ml

```diff
@@ -31,21 +31,21 @@ let test_tainting (lang : Lang.t) (file : Fpath.t) options config def =
       (Dataflow_tainting.mk_empty_java_props_cache ())
       def
   in
-  let taint_to_str taint =
-    let show_taint t =
-      match t.Taint.orig with
-      | Taint.Src src ->
-          let tok1, tok2 = (fst (Taint.pm_of_trace src.call_trace)).range_loc in
-          let r = Range.range_of_token_locations tok1 tok2 in
-          Range.content_at_range file r
-      | Taint.Arg arg -> Taint._show_arg arg
-      | Taint.Control -> "<control>"
-    in
-    taint |> Taint.Taint_set.elements |> List_.map show_taint
-    |> String.concat ", "
-    |> fun str -> "{ " ^ str ^ " }"
-  in
-  DataflowX.display_mapping flow mapping (Taint_lval_env.to_string taint_to_str)
+  (* let taint_to_str taint =
```

Comment:
at least add a comment why this code is commented now.

Reply:
I opened the PR mainly to run the Argo benchmarks, but had a few things to clean up like this one. I just tweaked `Taint.show_taints` and removed the commented out code here.